pkg/virt-controller/watch/workload-updater/workload-updater_test.go

```diff
@@ -291,8 +287,7 @@ var _ = Describe("Workload Updater", func() {
 				newVirtualMachine(fmt.Sprintf("testvm-up-to-date-non-migratable-%d", i), false, expectedImage, vmiSource, podSource)
 			}
 
-			// wait for informer to catch up since we aren't watching for vmis directly
-			time.Sleep(1 * time.Second)
+			waitForNumberOfInstancesOnVMIInformerCache(controller, 102)
```

Comment:
ditto; I'd appreciate the counter here, instead of counting mentally 1 + 1 + 50 + 50.

Reply:
Done.